synapse/http/client.py

```diff
@@ -114,19 +114,23 @@ def send_request():
             raise e
 
     @defer.inlineCallbacks
-    def post_urlencoded_get_json(self, uri, args={}):
+    def post_urlencoded_get_json(self, uri, args={}, headers=None):
         # TODO: Do we ever want to log message contents?
```

Comment:
No doc comment?

Reply:
ugh I got bored of writing them for existing functions. Suppose I should toughen up.

Reply:
Not that I'm in a position to complain, just wondered if it was accidental :)